mgr/manager.go

```diff
@@ -515,20 +620,33 @@ func (m *Manager) Server(sync bool) {
 	// NB: this loop could be more intelligent; for example,
 	// updating the next expiration independently of checking
 	// certificates.
+
+	metrics.ManagerInterval.WithLabelValues(m.Dir, m.Interval).Set(1)
 	go m.ProcessQueue()
 
 	if sync {
 		failed := m.CheckCertsSync()
 		if failed != 0 {
 			log.Infof("manager: failed to provision %d certs (certs are queued)")
 		}
+		m.CheckDiskPKI()
 	} else {
 		m.CheckCerts()
+		m.CheckDiskPKI()
```

Comment:
It looks like it'd have the same effect just to `m.CheckDiskPKI()` once right below this if/else.

Reply:
You're right, I'll change this.